test2.js

```diff
@@ -0,0 +1,31 @@
+import { Selector } from 'testcafe';
+import MainPage from './pages/mainPage.js';
+import CommunityRecomendationPage from './pages/communityRecomendationPage.js';
+import { getRandomIntInclusive } from './utils/helpers.js';
+
+fixture`Getting Started`
+    .page`https://store.steampowered.com/`;
+
+test('Playtime filter test', async t => {
+    await t.expect(Selector(MainPage.logo).exists).ok({ timeout: 3000 });
+
+    await MainPage.openCommunityRecommendations();
+
+    await t.expect(Selector(CommunityRecomendationPage.pageTitle).innerText).eql('THE COMMUNITY RECOMMENDS');
```

Comment:
this string value can be stored in a constant

Reply:
moved to constant